rocAL/rocAL_pybind/amd/rocal/types.py

```diff
@@ -34,7 +34,9 @@
 from rali_pybind.types import NHWC
 from rali_pybind.types import NCHW
 
-
+#     RaliDecodeDevice
```

Comment:
is the empty file above indented? Please remove

Reply:
Yes Rajy
The init.py files are required to make Python treat the directories as containing packages
In the simple case, they can also be empty.
[init.py files can be empty](http://docs.python.org/tutorial/modules.html#packages)